### ISSUE TITLE
languages_mapping does not work for sublanguages

### ISSUE THREAD

**Issue by gregnolle** [crowdin/crowdin-cli-2#59]
For example, if I define mappings for zh-CN and zh-TW, it uses the value I specify for zh-TW for both. This same configuration with v1 of the CLI works correctly. Example config:

```
"files": [
  {
    "source": "/path/to/messages.properties",
    "translation": "%original_path%/%file_name%_%two_letters_code%.%file_extension%",
    "languages_mapping": {
      "two_letters_code": {
        "zh-CN": "zh_CN",
        "zh-TW": "zh_TW"
      }
    }
  }
]
```

**Comment by ihor-popyk** [comment]
Hi @gregnolle 
Fixed this bug in 2.0.18
Sorry for this problem.
#60

**Comment by gregnolle** [comment]
Confirmed working now in 2.0.18. Thanks for fixing so quickly!
